CHANGELOG.md

```diff
@@ -3,6 +3,17 @@
 This file lists the main changes with each version of the Fyne toolkit.
 More detailed release notes can be found on the [releases page](https://github.com/fyne-io/fyne/releases). 
 
+## 2.1 - Ongoing
+
+### Added
+
+### Updated
+
+* Focusable widgets are no longer focused on tap, add canvas.Focus(obj) in Tapped handler if required
```

Comment:
This would also mean that custom widgets that are focused when tapped cannot be extended? (Because .super() is private? 🤔)

Reply:
Yes, thanks - I missed some elements here and have tidied it all up